gateway/utils/graphql/read.go

```diff
@@ -12,27 +12,27 @@ import (
 	"github.com/spaceuptech/space-cloud/gateway/utils"
 )
 
-func (graph *Module) execLinkedReadRequest(ctx context.Context, field *ast.Field, dbType, col, token string, req *model.ReadRequest, store utils.M, loader *loaderMap, cb dbCallback) {
+func (graph *Module) execLinkedReadRequest(ctx context.Context, field *ast.Field, dbType, col, token string, req *model.ReadRequest, store utils.M, cb dbCallback) {
 	// Check if read op is authorised
 	actions, _, err := graph.auth.IsReadOpAuthorised(ctx, graph.project, dbType, col, token, req)
 	if err != nil {
 		cb("", "", nil, err)
 		return
 	}
 
-	dataLoader := loader.get(getFieldName(field)+"."+store["path"].(string)+".linked."+col, graph)
+	// dataLoader := loader.get(getFieldName(field)+"."+store["path"].(string)+".linked."+col, graph)
 
 	go func() {
-		// Create dataloader key
-		key := model.ReadRequestKey{DBType: dbType, Col: col, HasOptions: false, Req: *req}
-		result, err := dataLoader.Load(ctx, key)()
+		req.Options.Prefix = fmt.Sprintf("%s.%s.linked.%s", getFieldName(field), store["path"].(string), col)
```

Comment:
Prepend a `ksuid` to the prefix you are generating here. 

**You will not generate the ksuid here or it will beat the purpose**.

The `ksuid` will be generated when the request is being made and will be passed down till here. **Don't store the KSUID in the graph module struct**. The validity of a KSUID should be a for a single request only